backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -50,12 +57,29 @@ public void setUp(){
         inMemoryVirtualQueueRepository.getShopOwnerMap().clear();
     }
 
+    private UUID addShopToRepository(){
```

Comment:
Nit. All helper methods should go in the end after all the tests.

Reply:
done!